.github/workflows/main.yml

```
@@ -1,26 +1,121 @@
+          echo "new_comments=$(jq length delta.json)" >> "$GITHUB_OUTPUT"
+
+      - name: Generate Action Plan using OpenAI CLI
+        if: steps.diff.outputs.new_comments != '0'
+        env:
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4o" # Optional: defaults to "gpt-4"
-          exclude: "**/*.json, **/*.md" # Optional: exclude patterns separated by commas
\ No newline at end of file
+        run: |
+          cat <<EOF > prompt.txt
+Act like a highly skilled engineering lead and product-quality reviewer. You will refine and iterate an action plan based on newly added PR comments and the last recorded plan.
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 77: could not find expected ':'

(syntax-check)